.travis.yml

```diff
@@ -28,11 +33,15 @@ before_script:
 - java -jar elasticmq-server-0.14.2.jar &
 before_install:
 - sudo rm -f /etc/boto.cfg
+- export CHROME_BIN=chromium-browser
+- export DISPLAY=:99.0
+- sh -e /etc/init.d/xvfb start
 install:
 - pip install -r requirements/dev.txt
 - pip install awscli==1.16.57 coveralls
 script:
 - flake8 ./
+- node_modules/karma/bin/karma start karma.conf.js --single-run
```

Comment:
Can we please install `karma-cli` in order to avoid this long command. For reference: http://karma-runner.github.io/4.0/intro/installation.html

Reply:
Please do the same for running tests in nodejs docker container.